fe/src/project/AspNetCore/areas/Orders/components/NewOrderForm/NewOrderForm.js

```diff
@@ -89,7 +89,7 @@ function NewOrderForm(props) {
             sku: product.sku,
             name: product.name,
             imageId: product.images ? product.images[0] : null,
-            quantity,
+            quantity: parseInt(quantity),
```

Comment:
**suggestion:** let's make it int instead of double in the BE. We can avoid the parseInt here thererafter